### ISSUE TITLE
Proposed bugfix for "seeElement" (nightmare)

### ISSUE THREAD

**Issue by markmarijnissen** [Codeception/CodeceptJS#641]
```javascript
I.seeElement("#header")
```

Using CodeceptJS v1.0.0 with Nightmare.

While the element is visible, I receive an error with an incorrect stacktrace:

```
  expected number of elements on a page '0' not to equal '0'

  Scenario Steps:
  
  - I.executeScript(function()) at Test.<anonymous> (d1.login2.test.js:77:26)
  - I.dontSeeInCurrentUrl("diagnose/list") at Test.<anonymous> (d1.login2.test.js:76:5)
  - I.waitFor(function()) at logout (d1.login2.test.js:66:5)
  - I.click("#menuLogout") at logout (d1.login2.test.js:65:5)
  - I.click("#header button") at logout (d1.login2.test.js:64:5)
  - I.seeElement("#header") at logout (d1.login2.test.js:63:5)
  - I.click("button#login") at Object.login (steps_file.js:10:12)
  - I.fillField("input#password", "xxx") at Object.login (steps_file.js:9:12)
  - I.fillField("input#username", "xxx") at Object.login (steps_file.js:8:12)
  - I.amOnPage("/#/diagnose/list") at Test.<anonymous> (d1.login2.test.js:73:5)
```

When inspecting the source code and element, I notice the element `offsetParent` is null. The `seeElement` ignores those elements, see https://github.com/Codeception/CodeceptJS/blob/master/lib/helper/Nightmare.js#L397

When I look at the nightmare implementation of visible, they check for `offsetWith` and `offsetHeight`, see https://github.com/segmentio/nightmare/blob/master/lib/actions.js#L72

This seems to me a more reliable method.

As a workaround, I now use a custom helper:
```javascript
'use strict';

class SeeElement extends Helper {
  seeElement(locator) {
    this.helpers['Nightmare'].browser.visible(locator);
  }
}

module.exports = SeeElement;
```

**Comment by DavertMik** [comment]
Looks like you have fixed this issue. Could you patch the Nightmare helper and write a failing test for this case? If writing test seems hard just patch it and send PR and I will finish it

**Comment by reubenmiller** [comment]
Fixed in v1.1.6 which was release today.

Updated the behaviour as suggested by this ticket
